README.md

```diff
@@ -59,17 +40,37 @@ AltStore and AltServer are both fairly straightforward to compile and run if you
-6. Build + run app! 🎉
 
-## Licensing
+5. **(SideStore app only)** Change the value for `ALTDeviceID` in the Info.plist to your device's UDID. Normally, SideStore embeds the device's UDID in AltStore's Info.plist during installation. When running through Visual Studio you'll need to set the value yourself or else Sidetore won't resign (or even install) apps for the proper device. You can achieve this by changing a few things to be able to build and use SideStore.
+
+**Steps for making SideStore run with your own build**
+This is all in vscode because Xcode UI is tricky
+
+AltBackup/AltBackup.entitlements
+Change the app group to something you can sign like group.com.[Rick].SideStore
 
-Due to the licensing of some dependencies used by AltStore, I have no choice but to distribute AltStore under the **AGPLv3 license**. That being said, my goal for AltStore is for it to be an open source project *anyone* can use without restrictions, so I explicitly give permission for anyone to use, modify, and distribute all *my* original code for this project in any form, with or without attribution, without fear of legal consequences (dependencies remain under their original licenses, however).
+AltStore/AltStore.entitlements
+Same thing ^^
 
-## Contact Me
+AltWidget/AltWidgetExtension.entitlements
+Same thing ^^
 
-* Email: riley@rileytestut.com
-* Twitter: [@rileytestut](https://twitter.com/rileytestut)
+AltWidget/Info.plist
+Change ALTAppGroups to your group
+
+AltStore/Info.plist
+Change ALTAppGroups to your app group name group.com.[Rick].SideStore
+Change ALTDeviceID to your device's UDID. You can fetch it with libimobiledevice if needed.
+
+AltStore/project.pbxproj
+Change all the DEVELOPMENT_TEAMs to your dev team ID. It's what AltStore appends to your bundle IDs.
+Change all the PRODUCT_BUNDLE_IDENTIFIERs to something you can sign like com.[Rick].SideStore...
+
+Shared/Extensions/Bundle+AltStore.swift
+Change the string "group.com.rileytestut.AltStore" to your group you are using.
+
+Build + run app! 🎉
+
+## Licensing
 
-Questions about AltStore in general? Make sure to read the FAQ at https://altstore.io/faq/
+Due to the licensing of some dependencies used by SideStore, I have no choice but to distribute SideStore under the **AGPLv3 license** because this is a fork of Altstore. That being said, our goal for SideStore is for it to be an open source project *anyone* can use without restrictions, so we explicitly give permission for anyone to use, modify, and distribute all *our* original code for this project in any form, with or without attribution, without fear of legal consequences (dependencies remain under their original licenses, however).
```

Comment:
Maybe just state that it's licensed under AGPLv3. I don't like the tone of this.

Reply:
Changed in latest commit